.snippets/code/tutorials/polkadot-sdk/build-a-blockchain/add-trusted-nodes/key-sr25519-1.html

```diff
@@ -0,0 +1,13 @@
+<div id='termynal' data-termynal>
+    <span data-ty="input"><span class="file-path"></span>./target/release/solochain-template-node key generate \</span>
+    <span>--scheme Sr25519 \</span></br>
+    <span>--password-interactive</span>
+    <span data-ty>Secret phrase: digital width rely long insect blind usual name oyster easy steak spend</span>
```

Comment:
```suggestion
    <span data-ty>Key password:</span>
    <span data-ty>Secret phrase: digital width rely long insect blind usual name oyster easy steak spend</span>
```

For reference
<img width="637" alt="Screenshot 2024-10-08 at 2 10 19 AM" src="https://github.com/user-attachments/assets/bfec33a4-a246-416b-9088-15d5561d8298">